### ISSUE TITLE
Publish Window Size Property

### ISSUE THREAD

**Issue by blackalkane** [amqp/rhea#398]
Hello @grs 

Does rhea has a publish window size property to set for a connection/session/sender, it's a property that limits the maximum number of messages that can be published before the broker must receive an acknowledgement from the appliance?

We're using rhea as our AMQP client link to our internal service called - Message-queuing (Message Broker as a Service @ SAP). It's based on Solace PubSub. For application using other protocols like JCSMP, they can configure it easily https://docs.solace.com/API/API-Developer-Guide/Sending-Guaranteed-Messages.htm#publishing_guaranteed_messages_3926332939_617205

Thank you.

**Comment by grs** [comment]
No. There isn't really anything in the AMQP protocol that would allow that information to be conveyed by the sender to the receiver. The window is always controlled by the receiving side.

**Comment by grs** [comment]
You could ask the broker/service support how they determine that publish window in the AMQP case. If there is anything the sender can control to influence that which I am missing we can then determine how it is exposed through rhea.
